### DISCUSSION TITLE
Assistance Needed with Educational Package Request Rejection

### DISCUSSION THREAD

**Discussion by msabeq94** [community/community#131242] [GitHub Education]
### Hi everyone,

Hello GitHub Education Support Team,

My name is Ibrahim, and I am currently a student at Cairo University academic year 2023/2024 from Egypt  I recently applied for the GitHub educational package but my request was rejected. I am writing to seek assistance in understanding why my application was rejected and how I can successfully obtain the educational package.

Here is a summary of the steps I have taken and the information I have provided:

1- I uploaded my student ID picture.
2- I updated my GitHub profile, including my bio and billing information.
3- I validated my university email address.
4-I uploaded my student ID along with translation of the text on the ID.

Despite providing all the required information and ensuring that my documents meet the specified criteria, my request was still rejected. I believe I am eligible for the educational package and would appreciate any guidance on what additional steps I need to take or any specific details I need to provide to resolve this issue.

Thank you for your time and assistance.

**Comment by RajeevRanjan5642** [comment]
Hey 👋 I think I can help u with that since I have availed the github education pack 3-4 days ago only. First time my request was rejected since I had not secured my github account using two-factor-authentication. As soon as I did it and requested again ,my request was approved immediately. So, I guess that might be the case with you. Below I am attaching snapshot of my case. If my answer helps, please mark it as answer, so that others can get help from it too 💖.
![Screenshot 2024-07-06 205033](https://github.com/community/community/assets/95275538/1acb3746-fea4-4fad-84fe-950368a69562)

**Comment by msabeq94** [comment]
i am using two-factor-authentication

**Reply by RajeevRanjan5642** [reply]
Ok
I am attaching a snapshot below, please make sure u have completed all the points mentioned in **Before you begin section**.
![Screenshot 2024-07-07 073434](https://github.com/community/community/assets/95275538/810ed7db-ec6e-4fb1-b8b0-5d705f97f25c)
Hope it helps 💖!!
